### ISSUE TITLE
Gauze should not stop all bleeding at once 

### ISSUE THREAD

**Issue by K-Dynamic** [space-wizards/space-station-14#33020]
## Description

A single unit of gauze will stop ALL bleeding no matter how much you actually have. If anything, it should be similar to cauterisation in which it requires multiple uses.

Tourniquets are basically overshadowed by gauze (it doesn't help you can't stack them or obtain them outside of security but that's another issue).

Not sure if the same logic should be applied to sutures as they're much rarer.

**Reproduction**

1. Obtain 1 unit of gauze.
2. Stab yourself multiple times until near crit.
3. Apply gauze.
4. Magically stop all bleeding (you will still die of bloodloss unless you have saline syringes).

**Screenshots**

![image](https://github.com/user-attachments/assets/f2994095-ef98-4165-85ce-96f804b2e5fe)

**Comment by Aisu9** [comment]
> Keep in mind I opened this issue in the face of #32744 being approved, which will make gauze much easier to obtain.

only easier for medical, and crafting faster doesn't mean easier to obtain. also tourniquet are very much a medical items(distributed to security) and not the other way

**Comment by superjj18** [comment]
Is people bleeding to death despite having gauze on hand really that important?

I’d rather nerf the speed of self-clotting necessitating some form of treatment

**Comment by CookieMasterT** [comment]
You can make all brute healing sources fix bleeding and make gauze no longer do this magic, would be reasonable and consistent.

**Comment by GoodWheatley** [comment]
The upside of bringing a tourniquet is that it applies near instantly when you use it, compared to gauze taking several seconds to stop bleeding. You can still deal with bleeding faster through cauterization compared to the time to use gauze, or you could inject yourself with an emergency medipen so the tranexamic acid in it stops your bleeding.

**Comment by Ilya246** [comment]
> The upside of bringing a tourniquet is that it applies near instantly when you use it, compared to gauze taking several seconds to stop bleeding. You can still deal with bleeding faster through cauterization compared to the time to use gauze, or you could inject yourself with an emergency medipen so the tranexamic acid in it stops your bleeding.

yeah just wanted to say this, gauze takes a 15 year (relative to how fast you bleed) doafter to use so gauze only taking one use for all bleeding makes sense, whereas a tourniquet is basically instant

from testing:
- gauze: the best, 5 seconds to stop all bleeding and heals 10 piercing
- tourniquet: stops your bleeding in around 1 second, but it damages you and this causes you to slightly bleed again
- cautery: the worst option, 8 seconds to stop full bleeding and damages you even more than tourniquet (40 heat damage)

so, my conclusion? keep gauze as-is, but instead buff tourniquet and definitely buff the cautery to maybe at most deal 15 heat to you when stopping full bleed

**Comment by GoodWheatley** [comment]
The cautery actually does something??? I just hit myself with a welding tool

**Comment by Ilya246** [comment]
proposal:
- cautery: make it deal less heat and have it directly reduce your bleeding so that it takes ~8 seconds and does 15 heat to stop full bleed
- tourniquet: rework it, have it be something that "stays applied" to you

tourniquet rework details:
- application speed stays as-is
- while applied, you are unable to use a hand
- while applied, stops bleeding from reducing your blood level
